packages/web3-utils/test/fixtures/converters.ts

```diff
@@ -267,3 +267,15 @@ export const toCheckSumValidData: [string, string][] = [
 	['0x5fbc2b6c19ee3dd5f9af96ff337ddc89e30ceaef', '0x5FBc2b6C19EE3DD5f9Af96ff337DDC89e30ceAef'],
 	['0xa54D3c09E34aC96807c1CC397404bF2B98DC4eFb', '0xa54d3c09E34aC96807c1CC397404bF2B98DC4eFb'],
 ];
+
+export const bytesToBufferInvalidData: [any, string][] = bytesToHexInvalidData;
+
+export const bytesToBufferValidData: [Bytes, Buffer][] = [
+	[new Uint8Array([72]), Buffer.from('48', 'hex')],
+	[new Uint8Array([72, 12]), Buffer.from('480c', 'hex')],
+	['0x9c12', Buffer.from('9c12', 'hex')],
+	['0X12c6', Buffer.from('12c6', 'hex')],
+	['0X1', Buffer.from('01', 'hex')],
```

Comment:
```suggestion
	['0X1', Buffer.from('01', 'hex')],
	['0x1', Buffer.from('01', 'hex')],
	['0x0', Buffer.from('00', 'hex')],	
    ['0X0', Buffer.from('00', 'hex')],	
    ['1234', Buffer.from('1233', 'hex')],	
```

Reply:
Also add these invalid use cases as well. 

```
    ['1', Buffer.from('01', 'hex')],	    
    ['0', Buffer.from('00', 'hex')],	    

```

Reply:
Add some of them, but, made some changes

As valid I added:
['0x1234', Buffer.from('1233', 'hex')] instead of ['1234', Buffer.from('1233', 'hex')]

and in invalids:
['1', Buffer.from('01', 'hex')],	    
['0', Buffer.from('00', 'hex')] 
I changes the `Buffer.from` to the error message.
Are these changes ok?

Reply:
Sure